### ISSUE TITLE
A11y_Model Builder_Create New Experiment_Keyboard: Keyboard focus is not going to close(x) button in 'Create New Experiment' dialog

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [dotnet/machinelearning-modelbuilder#1570]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**
#A11y_Model Builder; #A11yAuto; #TabStops; #A11yMAS; #A11yTCS; #WCAG2.4.3; #Focus Order; #Keyboard; #Win32; #DesktopApp; #A11ySev3; #ModelBuilder-Win32-Jun2021; #Visual Studio Client;

### Environment Details:
Application name: Model Builder

### OS Details:
Windows Version 21H1 OS Build (19043.1083)
Microsoft Visual Studio 

### Note:
This issue is observed in all screens i.e., Create New Workspace, Create new Resource Group, Create new Compute dialogs.

### Repro Steps:

1. Enable ML.NET Model builder as a preview feature in tools>options>Environment>preview features.
2. Create a .NET C# project and tab till project under solution explorer and Add machine learning.
3. Tab till Image classification under scenario tab and press enter.
4. Select Azure under environment tab and tab till set up workspace and press enter.
5. In 'Create New Experiment' dialog verify whether focus is going to close button or not

### Actual Result:
Keyboard focus is not going to close(x) button in 'Create New Experiment' dialog

### Expected Result:
Keyboard focus should go to close(x) button in 'Create New Experiment' dialog. All the interactive controls in the dialog should receive keyboard focus and should be keyboard accessible.

### User Impact:
Keyboard users will find it difficult if close button is not accessible.

**Comment by beccamc** [comment]
This seems to be the fault of the VSDialog component we are using. Model Builder doesn't have control of the "X" button. I see similar behavior in the Dialog control, that isn't VS specific. 

**Action**: Need to email the WPF accessibility team.

**Comment by beccamc** [comment]
@zewditu Can you get the accessibility team help with this one too?

**Comment by zewditu** [comment]
Sure!

**Comment by beccamc** [comment]
@NiharikaTipirishetty I've checked with the team that owns this control. They said... 

> The close button is usually part of a window’s non-client area, which doesn’t typically participate in the tab loop.  Instead, windows have a system menu, accessible with Alt+Space.  That menu provides a Close command.

If you press Alt+Space you'll see a menu that is accessible. 

![image](https://user-images.githubusercontent.com/9122518/130513918-bc72ac04-3404-4da7-b13c-6e3895b52421.png)

**Comment by NiharikaTipirishetty** [comment]
On pressing Alt + space, a popup will be displayed for keyboard users where close option is available to close the dialog. As there is an alternative for keyboard users to close the dialog, closing this issue.

https://user-images.githubusercontent.com/83570638/131445765-840f6340-1ed7-48a0-bd4a-63f854e79474.mp4
